tests/cypress/forms_spec.js

```diff
@@ -98,11 +104,27 @@ context("Marketo dynamic form", () => {
 
     getIframeBody().find(".rc-anchor-content").click();
 
-    cy.wait(10000); // eslint-disable-line
+    cy.wait(3000); // eslint-disable-line
     cy.get("#mktoForm_1251")
       .submit()
       .should((page) => {
-        expect(page[0].action).to.equal("/marketo/submit");
+        expect(page[0].action).to.include("/marketo/submit");
       });
+    cy.get(".p-modal__close").click();
```

Comment:
Might be better to use something that's visible to the user instead of a class name. I think `Close active modal` aria-label would work (something like `cy.findByLabelText("Close active modal")`.

Reply:
You can find some good examples of how you can write cypress tests in conjunction with @testing-library/cypress here: 

- https://testing-library.com/docs/cypress-testing-library/intro/#examples 
- https://github.com/testing-library/cypress-testing-library/blob/97939da7d4707a71049884c0324c0eda56e26fc2/cypress/integration/find.spec.js

Reply:
Great - Thanks @petermakowski